.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @bajtos @juliangruber @pyropy @NikolasHaimerl
```

Comment:
Please mirror existing projects for consistency, since we have already fixed bugs there: https://github.com/CheckerNetwork/spark-evaluate/commit/7548057f3c9609c4bc52baf896b0a85d7a7f8197

Reply:
Thanks for pointing this out. I wasn't sure what's the latest recommended setup, so I just copied CODEOWNERS from a random repo I found checked out on my drive.

Reply:
Addressed in [62c8051](https://github.com/CheckerNetwork/node/pull/711/commits/62c8051460dcac9b2fab4a337cce40a397527d4f)

Reply:
> Thanks for pointing this out. I wasn't sure what's the latest recommended setup, so I just copied CODEOWNERS from a random repo I found checked out on my drive.

You could do a code search for all code owners files in our org, and see which ones had the most recent commits https://github.com/search?q=org%3ACheckerNetwork%20CODEOWNERS&type=code